### PR TITLE
Prepare release v2021.1

### PR DESCRIPTION
**Pull request by matsduf**
Updates files updated by script:
* docs/specifications/tests/ImplementedTestCases.md
* docs/specifications/tests/README.md
* docs/specifications/tests/TestMessages.md

Updated file:
* Changes

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -1,5 +1,37 @@
 Release history of Zonemaster
 
+v2021.1 2021-05-31 (public release version)
+
+ [Features]
+ - Adds implementation of test cases DNSSEC15, DNSSEC16 and DNSSEC17 for
+   controll of CDS and CDNSKEY. See Zonemaster-Engine for this feature.
+ - Adds implementation of test case Basic04 and decreased serverity level
```
Comment (pnax) [inline Changes:8]: typo -> "severity"

**Review by pnax — APPROVED** [review]
Just a typo.

File: Changes (hunk 1 above)
Comment (matsduf) [reply Changes]: Fixed.

File: Changes [hunk 2]
```diff
@@ -1,5 +1,37 @@
 Release history of Zonemaster
 
+v2021.1 2021-05-31 (public release version)
+
+ [Features]
+ - Adds implementation of test cases DNSSEC15, DNSSEC16 and DNSSEC17 for
+   controll of CDS and CDNSKEY. See Zonemaster-Engine for this feature.
```
Comment (mattias-p) [inline Changes:7]: "Controll" is misspelled. Though I believe "checking" would be better than "control of".

**Review by mattias-p — APPROVED** [review]

File: Changes (hunk 2 above)
Comment (matsduf) [reply Changes]: Fixed. Pleaes review again.

**Review by mattias-p — APPROVED** [review]

File: Changes [hunk 3]
```diff
@@ -1,5 +1,37 @@
 Release history of Zonemaster
 
+v2021.1 2021-05-31 (public release version)
+
+ [Features]
+ - Adds implementation of test cases DNSSEC15, DNSSEC16 and DNSSEC17 for
+   checking CDS and CDNSKEY. See Zonemaster-Engine for this feature.
+ - Adds implementation of test case Basic04 and decreased severity level
+   of messages in other test cases to reduce duplicate massages. See
```
Comment (pnax) [inline Changes:9]: Didn't know Zonemaster was good at _massages_ ;-)
sorry I spotted another typo -> `messages`

**Review by mattias-p — APPROVED** [review]

File: Changes (hunk 3 above)
Comment (matsduf) [reply Changes]: Fixed.

**Review by pnax — APPROVED** [review]

**Pull request merged**